react/components/ImageLoader.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import ContentLoader from 'react-content-loader'
+
+const ImageLoader = () => (
+  <ContentLoader
+    style={{
+      width: '100%',
+      height: '100%',
+    }}
+    width={100}
+    height={56}
+    preserveAspectRatio="xMinYMin meet"
+  >
+    <rect width="100%" height="100%" />
```

Comment:
Can you use `className`  here? if you can, you should use `w-100` and `h-100` instead of `width="100%" height="100%"`